community/dash/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
```

Comment:
Can this be removed? Since Clang exists in `extra` and is a valid `$CC` option.

Reply:
I agree though we'd need to use another method of making builds static.

Reply:
`-static` is a `CFLAGS` option so it should be `export CFLAGS="$CFLAGS -static"` should one insist on making a static package.

Reply:
Lovely. I'm fine with this change. :+1:

@a-schaefers we may need to go back and fix any packages I may have merged which also set `CC`. I'm happy to do this myself if you like. It's my fault. :)

Reply:
let me see what I can do and get back to you. I'm very stubborn about keeping our base be it busybox/gnucoreutils/sbase/ubase/awk/gawk  etc.. all static tools with musl clib. Didn't think about clang.